package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -43,6 +43,7 @@
 """
 
 from __future__ import absolute_import
```

Comment:
Since you now use `zip` we also need to import the iterating zip from six. Please add the line `from six.moves import zip` below the __future__ import.